### ISSUE TITLE
Some assets don't display when built from git stk-code and svn stk-assets

### ISSUE THREAD

**Issue by EricxDu** [supertuxkart/stk-code#1682]
I cloned stk-code and checked-out stk-assets as per instructions. SuperTuxKart builds and executes. However a lot of assets dont ever display. Namely, when I enter any racecourse, only a few assets are visible: the skybox, the signal light (no bird), skid marks, most/all of the HUD, and some parts or markings on karts and other objects (but not the karts or objects themselves).

I showed stdout.log(1) to "vlj" on IRC and they said it looked normal, and that it's probably a bug in stk-assets.

1: http://pastebin.com/tSxjKEs6

**Comment by EricxDu** [comment]
Sorry I haven't followed up with screenshots yet. I had to find somewhere to upload, and now I can't remember where that was.

I'm back with a hunch, though. I've discovered I need to apt-get install a Debian package called libtxc-dxtn-s2tc0 for some textures to show up in two other games (Enemy Territory and Quake Wars). I am currently re-downloading and building stk from source to see if the problem persists.
